.bcr/metadata.template.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Could you add a README.md to this directory that explains each of the files, their format, and a link to the docs?